src/sentry/nodestore/bigtable/backend.py

```diff
@@ -168,6 +194,29 @@ def set(self, id, data, ttl=None):
         )
         self.connection.mutate_rows([row])
 
+    def get_multi(self, id_list):
+        if len(id_list) == 1:
+            id = id_list[0]
+            return {id: self.get(id)}
+
+        return {
+            id: future.result()
+            for id, future in izip(
+                id_list,
+                (
+                    self.thread_pool.submit(self.get, id)
+                    for id in id_list
+                )
+            )
+        }
+
+    def delete_multi(self, id_list):
+        if len(id_list) == 1:
+            return self.delete(id_list[0])
```

Comment:
nit: this sort of implies `self.delete` has a return value that gets discarded on the bulk operation — might make sense to split the `self.delete` and `return` into two separate statements